title_belt_nhl/main.py

```diff
@@ -141,7 +133,42 @@ def find_nearest_path(self, teams, path_string, from_date=None) -> str:
                 newTeams, path_string, cur_match.date
             )
         return path_string
-
+    
+    def is_game_complete(game: Game) -> bool:
+        """
+        Checks if the given `Game` is complete.  I *believe* the only enums are `FINAL` and `OFF`
+        per a quick check on previous completed season API responses
+        """
+        gameState = game['gameState'].upper()
+        return gameState == 'OFF' or gameState == 'FINAL'
+    
+    def determine_winning_team(game: Game) -> str:
+        homeScore = game['homeTeam']['score']
+        awayScore = game['awayTeam']['score']
+        if homeScore > awayScore:
+            return game['homeTeam']['abbrev']
+        elif awayScore > homeScore:
+            return game['awayTeam']['abbrev']
+        else:
+            return None
+        
+    def is_title_belt_game(game:Game, cur_belt_holder: str) -> bool:
+            return  game['homeTeam']['abbrev'] == cur_belt_holder or game['awayTeam']['abbrev'] == cur_belt_holder
```

Comment:
Seems like it would make sense to have these functions be the responsibility of the Game class.

Reply:
Or put the `Game` object into the `Match` class and have `Match` handle these?